packages/algob/src/lib/algo-operator.ts

```diff
@@ -60,4 +69,45 @@ export class AlgoOperatorImpl implements AlgoOperator {
       confirmedRound: txConfirmation[confirmedRound]
     };
   }
+
+  async deployASC (name: string, scParams: object, flags: ASCDeploymentFlags, payFlags: ASCPaymentFlags
+  ): Promise<ASCInfo> {
+    console.log("Deploying ASC:", name);
+    const result: ASCCache = await this.ensuredCompiled(name, false);
```

Comment:
`force` parameter is `false` but it should be taken from input. We don't have a global `--force` parameter but we may need to have it later. I think we should add a task to add the parameter.